openslides/agenda/static/js/agenda/site.js

```diff
@@ -123,32 +101,43 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
                 className: 'ngdialog-theme-default wide-form'
             });
         };
+        // open edit dialog
+        $scope.editDialog = function (item) {
+            var template, controller, resolve;
+            switch (item.content_object.collection) {
+                case 'core/customslide':
+                    template = 'static/templates/core/customslide-form.html';
+                    controller = 'CustomslideUpdateCtrl';
+                    resolve = { customslide: function() {
+                        return item.getContentResource().find(item.content_object.id) }};
+                    break;
+                case 'motions/motion':
```

Comment:
@ostcar Now, the modal edit dialog opens correctly via ui-route. But also the "background page" is  redirected to this edit url. How can I prevent this (the page should keep on agenda.list)?